infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -269,14 +283,22 @@ export class AtatWebApiStack extends cdk.Stack {
       }).fn
     );
 
-    this.restApi = apiGateway;
+    this.draftApi = draftApiGateway;
+    this.internalApi = internalApiGateway;
     this.addEmailRoutes(props);
     this.addTaskOrderRoutes(props);
     this.ssmParams.push(
-      new ssm.StringParameter(this, "ApiGatewayUrl", {
-        description: "URL for the API Gateway",
-        stringValue: apiGateway.urlForPath(),
-        parameterName: `/atat/${this.environmentId}/api/url`,
+      new ssm.StringParameter(this, "DraftApiGatewayUrl", {
+        description: "URL for the Draft API Gateway instance",
+        stringValue: draftApiGateway.urlForPath(),
+        parameterName: `/atat/${this.environmentId}/draftApi/url`,
+      })
+    );
+    this.ssmParams.push(
+      new ssm.StringParameter(this, "InternalApiGatewayUrl", {
+        description: "URL for the Internal API Gateway instance",
+        stringValue: draftApiGateway.urlForPath(),
+        parameterName: `/atat/${this.environmentId}/internalApi/url`,
```

Comment:
ahh nvm we've broken the configuration endpoint's ability to determine the API URL with this change. but I do like that we'll move to `internalApi`-only eventually which is more descriptive anyway